.config/nextest.toml

```diff
@@ -20,8 +20,8 @@ status-level = "skip"
 fail-fast = false
 # Retry failing tests in order to not block builds on flaky tests
 retries = 2
-# Mark tests as slow after 25mins, kill them after 50
-slow-timeout = { period = "1500s", terminate-after = 1 }
+# Mark tests as slow after 3 hours, kill them right after
+slow-timeout = { period = "3h", terminate-after = 1 }
```

Comment:
Do nightly tests really need 3h to run?

Reply:
I would expect them to take less , but I am giving some slack here so we don't hit another timeout again. Then we can tune accordingly.